WordPress/Classes/Models/BasePost.m

```diff
@@ -115,6 +123,18 @@ - (NSString *)statusTitle
     return [BasePost titleForStatus:self.status];
 }
 
+// If the post has a future publish date. This is different from "isScheduled" in that
```

Comment:
This wasn't meant to be a documentation comment, just an inline note.  
It wouldn't hurt to make it actual documentation so :+1: